### ISSUE TITLE
train from scratch then loss became nan

### ISSUE THREAD

**Issue by leonardodora** [hpcaitech/Open-Sora#292]
hi, when I train the t2v model from scratch, the loss became nan. I know it is important to have pretrained model like pixart. But it is hard to explained why the loss became nan if training the model from scratch.
![image](https://github.com/hpcaitech/Open-Sora/assets/13816743/ab4f2afc-ad78-4d03-9a5b-6f6c08ac275b)

**Comment by zhengzangw** [comment]
We do not encounter this problem. One potential possibility is the half-precision training. You should use bf16 instead of fp16.

**Comment by leonardodora** [comment]
> We do not encounter this problem. One potential possibility is the half-precision training. You should use bf16 instead of fp16.

Thanks, but bf16 is set in config file.  Did you train without pixart weights and the loss is not abnormal?

**Comment by zhengzangw** [comment]
Our computing resource is limited and does not try training from scratch for a long time.

**Comment by leonardodora** [comment]
> Our computing resource is limited and does not try training from scratch for a long time.

Thanks for your reply! btw, the newest update is awsome!

**Comment by kawayi12318** [comment]
@leonardodora 
I have the same problem. Have you solved it

**Comment by leonardodora** [comment]
> @leonardodora I have the same problem. Have you solved it

Not yet. Maybe a pixart need be retrained
